workarounds/skyrimvr.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+overwrite="$game_prefix/users/steamuser/AppData/Local/ModOrganizer/overwrite/SKSE"
+
+mkdir -p "$overwrite"
+
+cat << EOT >> "$overwrite/SKSE.ini"
+[Loader]
+RuntimeName=_SkyrimVR.exe
+EOT
```

Comment:
I missed a small detail in here when reading the original issue, my bad. This is not going to be portable, since the MO2 overwrite directory location may change depending on whether the user selects a portable or a global instance when running the application for the first time.

It's simple enough to get it working, though. What adding this file to the overwrite directory does is make MO2 add it to the virtual file system. We can achieve the same results by adding it to the real file system:

```suggestion
skse_config_dir="$game_installation/Data/SKSE"

mkdir -p "$skse_config_dir"

cat << EOT >> "$skse_config_dir/SKSE.ini"
[Loader]
RuntimeName=_SkyrimVR.exe
EOT
```